Composer/packages/server/src/models/settings/defaultSettingManager.ts

```diff
@@ -69,10 +69,11 @@ export class DefaultSettingManager extends FileSettingManager {
         maxImbalanceRatio: 10,
         maxUtteranceAllowed: 15000,
       },
-      skill: {
+      skillConfig: {
         isSkill: false,
         allowedCallers: ['*'],
       },
+      skill: {},
```

Comment:
Is there a reason to keep this around even though it's empty?

Reply:
Yeah. Its the object to which we append new skills from the UI. I presonally prefer to keep it so that when new bots are created by default it shows as an empty object and we just keep adding to the dictionary.